stock_vertical_lift_empty_tray_check/models/vertical_lift_operation_pick_zero_check.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import _, models
+
+
+class VerticalLiftOperationPickZeroCheck(models.TransientModel):
+    _name = "vertical.lift.operation.pick.zero.check"
+    _description = "Make sure the tray location is empty"
+
+    def _get_data_from_operation(self):
+        """Return picking, location and product from the operation shuttle"""
+        active_id = self.env.context.get("active_id")
```

Comment:
maybe a computed field like `current_move_line` (or `current_move`, not clear what you get here)